src/gradleinttest.sh

```diff
@@ -35,6 +42,7 @@ gradleinttest () {
     export IT_JADE_PEM_FILE_NAME=jade-dev-account.pem
     export GOOGLE_SA_CERT=jade-dev-account.pem
     export GOOGLE_CLOUD_PROJECT=${google_project}
+    export GOOGLE_CLOUD_DATA_PROJECT=${google_data_project}
```

Comment:
Would this be null for integration tests?

Reply:
I think it would an empty string, yes. But does that matter? I think no, because this env var is only read by the server process -- for connected tests, that's running locally, for integration tests, that's running in a separate container and the env vars there are set by the Helm charts.